machine.go

```diff
@@ -152,6 +152,14 @@ func (m *Machine) Logger() *log.Entry {
 	return m.logger.WithField("subsystem", userAgent)
 }
 
+// PID returns the machine's process ID
+func (m *Machine) PID() (int, error) {
```

Comment:
I could be missing something, but I don't see this function getting called anywhere.

Reply:
Yea, I wasn't sure about this function, but I assumed @jeromegn was using it for something, so I didn't touch it.

Reply:
I am using it, but it could be part of a separate PR. I need to get at the PID for the process for the scheduling software I'm using.

Reply:
@jeromegn Thanks! Can we move this to a separate PR?